### DISCUSSION TITLE
Why is KRE payout a fixed amount?

### DISCUSSION THREAD

**Discussion by bs9000** [kinfoundation/kin-rfcs#128] [KRE Suggestions]
I admit that I may not understand all the details of KRE 3.1, so maybe my question isn't valid. 

But here are my thoughts. 

I understand that the dayli payout is lowered to 220M. But why is the payout fixed?

Why isn't it dynamic (maybe within a range)?

So, if the apps are doing very well to get new users, raising the spend amounts and the spend value then KRE payout will increase. If the apps are doing nothing but maintaining the existing user base the payment will lower.

Currently KRE needs to find a way to distribute 220M KIN every single day - also if no app has actually done anything that would help KIN reaching the next level.

I just think I would be okay if KRE pays out less if there is no new value added.

What is new value? To me that is 
 - more spends than yesterday
 - more users than yesterday

It would be fair to get a payment for maintaining the users and spends you had yesterday. But there should be an additional "bonus" if you raise your numbers. And also a "punishment" if you can't maintain your numbers over time.

So instead of having a fixed KIN KRE amount what if a number of KIN was set for
 - maintain a user = x KIN
 - adding a user = x KIN
 - missing a user = -x KIN
 - raising spend = x KIN

And so on. But set a reward for actions that will drive KIN. The payment to the app will then be the sum of the  x's. 

Then a payment from KRE will only happen if the app did something for the KIN Ecosystem. Today if all the app are almost doing nothing for KIN still KRE needs to pay them 220M KIN.

Why?

**Comment by skyf0xx** [comment]
@bs9000 This is actually something we have considered in the past but reserved it perhaps for future versions. Ideally, the inflation of Kin should be proportional to the growth of the economy. If Kin is getting 1MM new users each day, and these new users are consuming 200MM of Kin for joining (200 Kin for each account), then the KRE should be dishing out 90% of the 200MM.

What this does is ensure that Kin is actually deflationary. The new users are demanding an extra 200MM which comes from exchanges, while the KRE would only be releasing ~180MM Kin to developers.

This is something we can consider as the size of growth of the economy increases (with new apps)

**Comment by skyf0xx** [comment]
There used to be a time when the KRE was totally dynamic in terms of Kin rewarded. However, we found that the amount of inflation being released into the system couldn't be controlled that way. Remember that even as you reward developers, they still have to sell Kin at some point to cover costs.


If you provide them with an unlimited supply, this leads to a continuous sell off. The only way to reward devs is to have a maximum limit they can compete for in one day. In this case, devs compete for 220 MM Kin each day. The dev with the highest users/ balances/ spends gets rewarded more than other devs.

**Comment by skyf0xx** [comment]
Ideally, the size of Kin's economy will one day be bigger than the amount of Kin coming from the KRE, which should make Kin deflationary in some ways.

**Comment by alexkruglov** [comment]
Understand these concerns @hitwill, but still a little worried that we are encouraging competition rather than collaboration among the apps to grow the overall pie

**Reply by skyf0xx** [reply]
Could you elaborate, @alexkruglov?

**Reply by alexkruglov** [reply]
Sure. I think the ecosystem is best off if we are indifferent where the earn or the spend happens. Key that wallets are earning and spending across the apps. So in a world where a user has one wallet (e.g., phantom), this KRE plan is not encouraging cross-app collaboration. There's one fixed pie and we are splitting it. If I get more, someone else gets less. This would give me pause about exposing other spend options to my users. And I think that's a bad thing

**Reply by Freedom-kin** [reply]
Hi Will,

I have to agree with  Alexkruglov and bs9000.

It seems strange that there is a set amount of KRE (220MM) no matter how apps are performing or whether they help grow the Kin Economy or not. There is a bag of Kin and it has to be divided, no matter what.  The KRE rules (either 3.0 or 3.1) seems to have only one purpose.
That is....we have a bag of money...how do we divide it among developers?
There seems to be no higher purpose.
 
Also whether there are 10 or 30 apps, the amount set stays the same.
I think the amount set should depend on the amount of apps and the extent to which they contribute to the Kin Economy.

This can simply be achieved by rewarding apps with a percentage of their net spends.
No other factors to include.

If you set it to 30% of their net SPENDS then the amount of KRE is variable and grows with the amount of apps.  The better they perform, the better they get paid because their net turnover gets higher and that's what they get rewarded the 30% for (of course with a cap).

No competition needed between developers. They all have the same chances.
If one app does very well it has no negative effect on other apps.
An additional advantage is that it promotes Economic Activity.
.
